src/domain/collection/eventList/EventList.tsx

```diff
@@ -39,13 +44,13 @@ const EventList: React.FC<Props> = ({ collection }) => {
       params: searchParams,
       sortOrder: EVENT_SORT_OPTIONS.END_TIME,
       superEventType: ['umbrella', 'none'],
-      eventType: 'event',
+      eventType: eventType ?? 'event',
     });
-  }, [locale, searchParams]);
+  }, [eventType, locale, searchParams]);
 
   const { data: eventsData, fetchMore, loading } = useEventListQuery({
     notifyOnNetworkStatusChange: true,
-    skip: !eventListQuery,
+    skip: !eventListQuery || !eventType,
```

Comment:
Maybe no need to invlude both  variables in this check?

Reply:
It's there because eventType can be undefined when the url is given, but it's a bad one.